experiments/list_unsupported.sh

```diff
@@ -25,7 +25,7 @@ include_path=""
 DIR=`dirname "$0"`
 file_name=$(basename "$1")
 
-for foldername in $(find ${path} -type d -name "*"); do
+for foldername in $(find ${path} -type d -name "*" | LC_ALL=posix sort ); do
```

Comment:
Can you add an explanation for what the `LC_ALL=posix` is for to the commit message?